Startup crash when Android 14 regional preferences are set to non-default values

Android 14 has introduced the ability to set so-called regional preferences for temperature units and the first day of the week ([\[1\]](https://support.google.com/android/answer/12395118?hl=en), [\[2\]](https://alexzh.com/regional-preferences-in-android-14/)). If I set any of those two settings (just one, or both) to a non-default value, Aard fails to start up. This happened both with 0.53 which I had originally installed (from my old phone's backup), as well as the new 0.55 which I've just installed.

```
--------- beginning of crash
12-29 16:23:32.662 23422 23422 E AndroidRuntime: FATAL EXCEPTION: main
12-29 16:23:32.662 23422 23422 E AndroidRuntime: Process: itkach.aard2, PID: 23422
12-29 16:23:32.662 23422 23422 E AndroidRuntime: java.lang.ExceptionInInitializerError
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.ibm.icu.util.ULocale.toLegacyKey(ULocale.java:3609)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.ibm.icu.util.ULocale$JDKLocaleHelper.toULocale(ULocale.java:4324)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.ibm.icu.util.ULocale$2.createInstance(ULocale.java:364)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.ibm.icu.util.ULocale$2.createInstance(ULocale.java:361)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.ibm.icu.impl.SoftCache.getInstance(SoftCache.java:69)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.ibm.icu.util.ULocale.forLocale(ULocale.java:429)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.ibm.icu.util.ULocale.<clinit>(ULocale.java:558)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.ibm.icu.text.Collator.getInstance(Collator.java:858)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at itkach.slob.Slob$KeyComparator.<init>(Slob.java:1031)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at itkach.slob.Slob$KeyComparator.<init>(Slob.java:1026)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at itkach.slob.Slob$Strength.<init>(Slob.java:228)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at itkach.slob.Slob$Strength.<init>(Slob.java:221)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at itkach.slob.Slob$Strength.<clinit>(Slob.java:205)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at itkach.aard2.BlobDescriptorList.<init>(BlobDescriptorList.java:64)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at itkach.aard2.BlobDescriptorList.<init>(BlobDescriptorList.java:53)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at itkach.aard2.Application.onCreate(Application.java:135)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1317)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7017)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2237)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:205)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:294)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8223)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:977)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: Caused by: java.util.MissingResourceException: Could not find the bundle com/ibm/icu/impl/data/icudt68b/keyTypeData.res
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.ibm.icu.impl.ICUResourceBundle.getBundleInstance(ICUResourceBundle.java:1205)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.ibm.icu.impl.locale.KeyTypeData.initFromResourceBundle(KeyTypeData.java:222)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	at com.ibm.icu.impl.locale.KeyTypeData.<clinit>(KeyTypeData.java:671)
12-29 16:23:32.662 23422 23422 E AndroidRuntime: 	... 27 more
```

--- a/exclude.txt
+++ b/exclude.txt
@@ -1566,7 +1566,6 @@ com/ibm/icu/impl/data/icudt68b/kde.res
 com/ibm/icu/impl/data/icudt68b/kde_TZ.res
 com/ibm/icu/impl/data/icudt68b/kea.res
 com/ibm/icu/impl/data/icudt68b/kea_CV.res
-com/ibm/icu/impl/data/icudt68b/keyTypeData.res
 com/ibm/icu/impl/data/icudt68b/khq.res
 com/ibm/icu/impl/data/icudt68b/khq_ML.res
 com/ibm/icu/impl/data/icudt68b/ki.res
@@ -3076,7 +3075,6 @@ com/ibm/icu/impl/data/icudt68b/th_TH_TRADITIONAL.res
 com/ibm/icu/impl/data/icudt68b/ti.res
 com/ibm/icu/impl/data/icudt68b/ti_ER.res
 com/ibm/icu/impl/data/icudt68b/ti_ET.res
-com/ibm/icu/impl/data/icudt68b/timezoneTypes.res
 com/ibm/icu/impl/data/icudt68b/tk.res
 com/ibm/icu/impl/data/icudt68b/tk_TM.res
 com/ibm/icu/impl/data/icudt68b/tl.res

